test/test_from_ndarray_c32_1d.f90

```diff
@@ -51,7 +51,7 @@ program test_from_ndarray_c32_1d_main
 
     if ( kokkos_is_initialized() ) then
       
-      ! call kokkos_print_configuration('flcl-test-', 'kokkos.out')
+      ! call kokkos_print_configuration('flcl-test-kokkos.out')
```

Comment:
Do you want to just get rid of these?
These are commented lines right?
One can just set the KOKKOS_PRINT_CONFIGURATION environment variable instead of uncomment and recompile